integrations/standalone/src/url-helper.ts

```diff
@@ -6,6 +6,15 @@ export namespace URLParams {
     return param !== null ? param : undefined;
   }
 
+  export function app(): string {
+    const serverUrl = server();
+    return parameter('app') ?? serverUrl.substring(serverUrl.lastIndexOf('/') + 1);
```

Comment:
use case of the serverUrl last part?

Reply:
today you can already change the server url (e.g. if the designer is started on 8082). And in this url the app is normally already included e.g. `localhost:8082/designer`

Reply:
If you like to support that please consider URL decoding for the app name

Reply:
Hm... honestly I don't think that it is really necessary. I thought it would be a nice idea, but I think you either use only a pid or then a pid with app and pmv...